analytics/analytics.go

```diff
@@ -207,21 +195,15 @@ func (dc *dataCollection) doPrepData() (*pb.SignedMetrics, error) {
 }
 
 func (dc *dataCollection) doSendData(sm *pb.SignedMetrics) error {
-	conn, cancel, err := dc.getGrpcConn()
+	config, err := dc.node.Repo.Config()
 	if err != nil {
 		return err
 	}
-	defer cancel()
-	defer conn.Close()
-
-	ctx, cancel := context.WithTimeout(context.Background(), callTimeout)
-	defer cancel()
-	client := pb.NewStatusClient(conn)
-	_, err = client.UpdateMetrics(ctx, sm)
-	if err != nil {
+	cb := cgrpc.StatusClient(config.Services.StatusServerDomain)
+	return cb.WithContext(context.Background(), func(ctx context.Context, client pb.StatusServiceClient) error {
```

Comment:
it's better to pass a context rather then context.Background(),
since we already have a context.